server/api/fixtures/initial.json

```diff
@@ -22,10 +22,10 @@
       "password": "pbkdf2_sha256$36000$N92TfR4PrqJ7$DZSbvw4wIWbDjrZ+xfDisnqCNwVoyoAdx/meV7pqkHY=",
       "last_login": "2018-09-09T00:35:53.883Z",
       "is_superuser": false,
-      "username": "mayowa.makinde",
-      "first_name": "Mayowa",
-      "last_name": "Makinde",
-      "email": "mayowa.makinde@andela.com",
+      "username": "olusola.oguntimehin",
+      "first_name": "Olusola",
+      "last_name": "Oguntimehin",
+      "email": "olusola.oguntimehin@andela.com",
```

Comment:
When committing code to a repo, it's advisable not to add actual data such as configurations or personal information. What's the purpose of adding this email and its user? I'm assuming this JSON file is just for pre-populating the database with information to test the application.

Reply:
@brianbirir The JSON file is to seed the database with information for testing purpose.